src/js/wrapper.ts

```diff
@@ -133,26 +149,28 @@ export const NATIVE = {
    * @param value string
    */
   setUser(user: User | null): void {
-    if (!this.isNativeClientAvailable()) {
-      throw this._NativeClientError;
-    }
+    if (this.enableNative) {
```

Comment:
Instead of wrapping everything in an `if` and therefore intend all the code below
you could do
```
if (!this.enableNative) {
  return; // or return Promise.reject(new SentryError("..."));
}
```